Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -144,6 +171,7 @@ public sealed class Server : ServerBase, IDisposable {
 
         public override async Task DidOpenTextDocument(DidOpenTextDocumentParams @params) {
             TraceMessage($"Opening document {@params.textDocument.uri}");
+            await _analyzerCreationTcs.Task;
```

Comment:
If there was an exception CreateAnalyzer, `_analyzerCreationTcs` will hold that exception. Is it correct to raise it every time API method is called?

Reply:
If analyzer creation failed, then the whole thing is broken. VSC restarts server automatically up to 3 times I believe.